src/go/k8s/tests/e2e/confluent-schema-registry/01-create-schema-registry.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: cp-schema-registry
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: cp-schema-registry
+  template:
+    metadata:
+      labels:
+        app: cp-schema-registry
+    spec:
+      containers:
+      - name: cp-schema-registry-server
+        image: "confluentinc/cp-schema-registry:6.0.1"
+        imagePullPolicy: "IfNotPresent"
+        command:
+        - /bin/bash
+        - -c
+        args:
+        - SCHEMA_REGISTRY_KAFKASTORE_BOOTSTRAP_SERVERS=schema-registry-test-0.schema-registry-test.$POD_NAMESPACE.svc.cluster.local:9092 /etc/confluent/docker/run
+        ports:
+        - name: schema-registry
+          containerPort: 8081
+          protocol: TCP
+        resources:
+          requests:
+            cpu: 600m
+            memory: 1Gi
```

Comment:
nit: are these resources are indeed required?

Reply:
Maybe not. Is there any problem with that value?

Reply:
I thought you can made the resources 100m for redpanda clusters specifically to not use many resources and have test parallelism. I'd guess similar thinking would apply here.

Reply:
This is schema registry no redpanda cluster. It runs on JVM. I saw that confluent helm by default generates `-Xms512M -Xmx512M` which I didn't want to change as I don't want to tune JVM.